__tests__/app.test.js

```diff
@@ -0,0 +1,45 @@
+const request = require('supertest');
+const app = require('../db/app');
+const db = require('../db/connection');
+
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data');
+
+beforeEach(() => {
+  return seed(data);
+})
+
+afterAll(() => {
+  return db.end();
+})
+
+
+describe("NC_NEWS", () => {
+    // get /api/topics
+    describe('/api/topics', () => {
+        test('GET 200: Responds with an array of topic object containing the following properties: slug and description.', () => {
+            return request(app)
+              .get('/api/topics')
+              .expect(200)
+              .then(({ body }) => {
+                const { topics } = body;
+                expect(topics).toHaveLength(3);
+      
+                topics.forEach(topic => {
+                  expect(typeof topic.description).toBe('string');
+                  expect(typeof topic.slug).toBe('string');
+                });
+            });
+        });
+
+        test("GET 404: Responds with an error if passed an invalid sort_by parameter.", () => {
```

Comment:
The test description says 'sort_by' issue but this is actually testing a wrong path or non-existent endpoint. Please correct this description. Also this test should be in a separate describe block, as it's not a test for the /api/topics endpoint.